_posts/2019-12-12-inspiration.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Inspiration
+date: 2019-12-12 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+If you ask someone who doesn't write often either of the following questions:
+
+- If you had to write something where do you find inspiration on?
+- What would you write about?
+
+They would probably say that they find inspiration on important event through their life, or that they find it in their daily routine or again that they would write about what they have already knowledge of.
+
+It might seem hard to find topics to write about at first, but with time you will find yourself with lot of ideas.
+
+The problem will change and become how to capture them effectively and how to know which ones to write about and which ones to discard.
+
+Some ideas are plainly bad, but for those kind of ideas you wouldn't even consider writing it down in your idea list so we are going to ignore those. For all the rest you can't know if an idea is good until you start experimenting with it. After writing a few short sentences it will become obvious if it's an idea that it worth pursuing or not.
```

Comment:
Bad ideas can still be good with proper execution

/R/ATBGE

Reply:
Not sure I wanted to see that.. xD but good point

Reply:
Added footnote